tests/data/RC101.vrp

```diff
@@ -0,0 +1,29 @@
+NAME: RC101
+COMMENT: Converted to TSPLIB by Accorsi & Cavaliere. Then converted to PyVRP's VRPLIB dialect.
```

Comment:
Maybe change the name (and file name) to something like "SmallVRPSPD.vrp", and update the comment to "Based on the RC101 file <converted to ....>.

It's a bit confusing that this is in fact, not, the RC101 instance. Just a small subset of it.